specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -464,559 +689,2050 @@
+                    }
+                }
+            },
+            "required": [
+                "value"
+            ]
+        },
+        "Case": {
+            "type": "object",
+            "description": "Represents a case in Azure Security Insights.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "properties": {
+                "etag": {
+                    "type": "string",
+                    "description": "Etag of the alert rule."
+                },
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Case data",
+                    "$ref": "#/definitions/CaseProperties"
+                }
+            }
+        },
+        "CaseProperties": {
+            "type": "object",
+            "description": "Describes case properties",
+            "properties": {
+                "lastUpdatedTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The last time the case was updated"
+                },
+                "createdTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The time the case was created"
+                },
+                "endTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The end time of the case"
+                },
+                "startTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The start time of the case"
+                },
+                "tags": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Tag"
+                    },
+                    "description": "List of tags"
+                },
+                "description": {
+                    "type": "string",
+                    "description": "The description of the case"
+                },
+                "title": {
+                    "type": "string",
+                    "description": "The title of the case"
+                },
+                "assignedTo": {
+                    "type": "string",
+                    "description": "The person the case is assigned to"
+                },
+                "severity": {
+                    "type": "string",
+                    "description": "The severity of the case",
+                    "enum": [
+                        "Critical",
+                        "High",
+                        "Medium",
+                        "Low",
+                        "Informational"
+                    ],
+                    "x-ms-enum": {
+                        "name": "severity",
```

Comment:
You have duplicate `x-ms-enum` types with different sets of values which is not allowed, see other definition [here](https://github.com/Azure/azure-rest-api-specs/blob/4cd8dd6c83ae0e8a06653691b3b6993568e81164/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json#L1599).  Either change the name or ensure both enums contain the same set of values.